docs/issues_catched.md

```diff
@@ -0,0 +1,34 @@
+
+# Issues Precaution catches
+
+Precaution utilizes multiple security static code analysis tools.
+Because of that the amount and types of checks for the different
+programming languages are different.
+The security static code analyzers are language-focused
+and some of the vulnerabilities they find are specific 
+for the concrete programming language.
+However, some of those rules can be grouped into a few groups:
+
+* Hardcoded credentials
+* SQL injections
+* Command injections
+* HTML injections
+* Weak random number generator
+* Imported/used weak cryptographic algorithms
+* Imported insecure libraries
+* Use of specific insecure functions from libraries
+* Weak key used for a cryptographic algorithm
+* Eval function used
+* Bind to all network interfaces example when using IP "0.0.0.0"
+* SSL/TLS turned off or bad versions used
+* Set bad file permission when creating
+* Poor file permissions used with chmod
+
+**Note:** Those vulnerabilities aims to warn the developers to audit the concrete piece of code.
```

Comment:
Sentence sounds awkward.  Why concrete?

Reply:
I will change the word but the idea is that we don’t want to warn the developers to audit the whole code base because then the linters are useless but for a specific code.